packages/@icon-magic/config-reader/src/config-loader.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+const importFresh = require('import-fresh');
+
+/**
+ * Loads a JavaScript configuration from a file.
+ * @param filePath The filename to load.
+ * @returns  The configuration object from the file.
+ */
+function loadJSConfigFile(filePath: string) {
+  console.debug(`Loading JS config file: ${filePath}`);
```

Comment:
Before we get too far into implementation, lets start using a logging utility. I'd recommend https://www.npmjs.com/package/debug